### ISSUE TITLE
Donation

### ISSUE THREAD

**Issue by tcsaddul** [jbaron/qx-typed#10]
Hi Peter (jbaron).  I would like to donate/sponsor **2k** USD as a way to thank you for this library.  I intend to donate more later. Can you setup a "Donate" or "Sponsor" button?

**Comment by jbaron** [comment]
That is very kind and generous, and I will look into setting up such a button and will let you know once available. Out of curiosity (if you can share this information of course), in what type of project did you use this library?

regards,
Peter

**Comment by tcsaddul** [comment]
This is our website:  https://www.digisoftph.com

Our flagship product "SIAS Online 3.x" (A more or less complete School Management System - From Curriculum Mgt, Admission, Enrollment, Learning Management, Class Record, Grading System) and of course other Management Systems like Human Resource and Payroll System, Supplies and Asset Mgt, Point of Sale, etc... (whew to many actually) were developed using your library.  Since your library is dependent on QooxDoo, I also offered the same donation/sponsor amount to QooxDoo team.

This is my way of thanking you and the qooxdoo team for these libraries enabled us to develop our systems.

For a quick lookup on some of our brochures:
https://digisoftph.com/downloads/SIAS%20Online%203.x%20Features.pdf
https://digisoftph.com/downloads/SIAS-LMS%20Onliine%203.x%20Features.pdf

![image](https://user-images.githubusercontent.com/2255528/104808220-b216c800-581f-11eb-8b93-6f2a9f808495.png)

I can say almost ALL browser-based UIs of these apps were developed using qx-typed.

Especially, the Pandemic has boosted our sales on our **Integrated** Leaning Management System.  Though most schools in the Philippines use Moodle, they encountered a lot of problems i.e. when more than 500 students are taking the exams simultaneously, moodle would crash, etc.  Moodle still uses a Server-side UI page generation which causes the server to crash on thousands of simultaneous users during synchronous exams.

**Comment by tcsaddul** [comment]
You can check on some of our online demos on youtube (NOTE: language used is Tagalog-English):
https://www.youtube.com/watch?v=LnqAG3jcmAE&t=23s
https://www.youtube.com/watch?v=6TF9N4TMqLA&t=150s
https://www.youtube.com/watch?v=ZYZUEVuPZBY&t=241s
and more... through my youtube channel.

**Comment by jbaron** [comment]
I added a donate button to the README file (so the home page of this github repository) that you can use for the donation. In case there are issues, just let me know. ****And again, thank you very much for your generosity. Much appreciated** !!!**

I also watched the videos that you shared, and it looks indeed like a very impressive and extensive piece of software you've created. Seems to cover the complete needs of a school, even larger universities. You even have the canteen covered :+1: . And a very impressive list of clients on your website.

P.S If you or your company want to be listed as a sponsor, just let me know. 

P.P.S I think it is still difficult today to beat qx for complex and data-entry focused web pages, especially when combined with TypeScript. When ever I have to deal with some other JS/TS web frameworks for similar type of work, I feel they are more fragile in nature.

**Comment by tcsaddul** [comment]
Thank you. I will send the donation tomorrow morning.  Tried a while ago but:

![image](https://user-images.githubusercontent.com/2255528/104937150-66634a80-59e8-11eb-80ff-dc99e38550b3.png)

**Comment by tcsaddul** [comment]
Indeed, qooxdoo is very powerful and super flexible for enterprise/business web apps like those that we have developed.  Personally, these are the apps that are currently needed in our country.  Though we focused our sales on schools, we are also now targeting other businesses like hospitals and government agencies.  In the Philippines, we have very few good online business web apps because mostly they still use the obsolete PHP backend page generation which is very slow and not scalable especially in schools with thousands of students accessing it especially now that face to face classes in schools are prohibited, the schools need to go online in their learning delivery.

**Comment by jbaron** [comment]
I just logged into the  paypal account and made sure everything is setup correctly. If the donate button still doesn't work tomorrow, please let me know and I'll try to find an alternative way.

Sounds like your business is doing very well and with such an impressive client list already, it should be possible to also get into other industries. 

BTW, I see the same here in The Netherlands: older products without modern web support and slow back-ends getting less popular. And companies are not afraid to invest into alternative solution. With people using impressive web applications on the internet every day, they are starting to expect the same from their software suppliers.

**Comment by tcsaddul** [comment]
Tried again at this hour (7:15 AM Philippines time) but same error appears. Will try again at 10am to ensure the time restriction is not being applied.

Our SIAS Online 3.x product line is doing really well.  We built our current success from our previous success with "SIAS Desktop 2.x" which was developed using Microsoft Visual FoxPro 9.0 in 2003.  As you can see in the List of Clients, we have clients who are still using our old product since 2003 and most of them (97%+) are still our clients. I can recall just 3 who went away due to reasons without our control (all of which treated our technical people badly so I cancelled the projects).  I have built our brand with good intentions and ensures the trust of our clients remain.  In fact, even those who already paid us 10+ years ago can still get FREE technical support from us while waiting patiently when they wish to upgrade to our Online version.  We also pay our technical people 5% of every project they handle in addition to their regular salaries and all their expenses are also refunded.

**Comment by tcsaddul** [comment]
Tried again at 11:00 Philippines time but I got the same error. I think we should try another way.  I use Credit Card and I know have used it in just a few weeks both in PayPal and Amazon so I know it works. I can also send payment through Western Union as a last resort.

**Comment by tcsaddul** [comment]
I called my bank and they told me there was an automatic fraud detection due to the large amount and they needed to adjust some parameters then asked me to try again and it was successful.

![image](https://user-images.githubusercontent.com/2255528/104986332-cb04c080-5a4d-11eb-88fd-326cea49c118.png)

Please confirm if you have received it.

**Comment by jbaron** [comment]
When I just woke up, indeed I saw the message from PayPal that confirmed the donation. **Thanks you very much**, both for the donation but also for the perseverance to even be able to make the donation !!!!!

I love the ethics that you and your company embrace. Both towards your customers but also the employees who work on the projects. After more than 20 years working for different companies as an employee, I started my own company about 1.5 years ago, and your approach is truly inspirational.
